build.sbt

```diff
@@ -129,6 +129,13 @@ lazy val core = libraryProject("core")
       ProblemFilters.exclude[DirectMissingMethodProblem]("org.http4s.parser.HttpHeaderParser.AUTHORIZATION"),
       ProblemFilters.exclude[DirectMissingMethodProblem]("org.http4s.parser.HttpHeaderParser.RANGE"),
       ProblemFilters.exclude[DirectMissingMethodProblem]("org.http4s.parser.HttpHeaderParser.CONTENT_RANGE"),
+      ProblemFilters.exclude[DirectMissingMethodProblem]("org.http4s.parser.HttpHeaderParser.X_B3_TRACEID"),
+      ProblemFilters.exclude[DirectMissingMethodProblem]("org.http4s.parser.HttpHeaderParser.X_B3_SPANID"),
+      ProblemFilters.exclude[DirectMissingMethodProblem]("org.http4s.parser.HttpHeaderParser.X_B3_PARENTSPANID"),
+      ProblemFilters.exclude[DirectMissingMethodProblem]("org.http4s.parser.HttpHeaderParser.X_B3_FLAGS"),
+      ProblemFilters.exclude[DirectMissingMethodProblem]("org.http4s.parser.HttpHeaderParser.X_B3_SAMPLED"),
+      ProblemFilters.exclude[IncompatibleTemplateDefProblem]("org.http4s.parser.ZipkinHeader"),
```

Comment:
I'm trying to keep this list in alphabetical order to reduce merge conflict pain.  That ordering includes the type of exclusion, so it's just a pipe through sort in $EDITOR.

Reply:
I hope I understood your comment correctly and the sorting now makes more sense